.github/workflows/maven.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - uses: actions/setup-java@v3
         with:
-          java-version: 8
+          java-version: 11
```

Comment:
Should we also update the Jenkinsfile?

Reply:
I update the Jenkins file to build with JDK 11, 17 and 21.